### DISCUSSION TITLE
pseudobulk differential expression analysis

### DISCUSSION THREAD

**Discussion by mitchTwoTimes** [GreenleafLab/ArchR#1093] [Questions / Documentation] [answered]
hello,

I know we can create pseudobulks for peak calling, but is there a way to repurpose these same pseudobulk samples for differential expression analysis? Would a potential workaround be to create a cell-level summarized experiment with `getGroupSE`, then make a pseudobulk on this object?

thanks!

**Comment by rcorces** [comment, marked as answer]
thats correct. We are working on making this part of a standard ArchR workflow as well but that will take a while due to limited bandwidth.

however, differential accessibility testing is already possible using a different (non-pseudobulk) workflow via `getMarkerFeatures()`. This is detailed in the manual. 
https://www.archrproject.com/bookdown/pairwise-testing-between-groups.html
